component-matrix/pom.xml

```diff
@@ -147,7 +147,7 @@
         <!-- Hardcoded in module.xml for the dual EE7 and EE8 support -->
         <version.org.glassfish.javax.json-1.0>1.0.4</version.org.glassfish.javax.json-1.0>
         <version.org.glassfish.soteria>1.0</version.org.glassfish.soteria>
-        <version.org.hibernate>5.1.13.Final</version.org.hibernate>
+        <version.org.hibernate>5.2.17.Final</version.org.hibernate>
         <version.org.hibernate.commons.annotations>5.0.2.Final</version.org.hibernate.commons.annotations>
         <version.org.hibernate.search>5.5.8.Final</version.org.hibernate.search>
```

Comment:
Probably `hibernate-seach` should be bumped to `5.7.x` or higher for compatibility with Hibernate 5.2.x. But which version to chose? The newest `5.9.1`?